packages/atomic/src/components/search/facets/atomic-popover/atomic-popover.tsx

```diff
@@ -0,0 +1,174 @@
+import {
+  buildSearchStatus,
+  SearchStatus,
+  SearchStatusState,
+} from '@coveo/headless';
+import {Component, h, Listen, State, Element} from '@stencil/core';
+import {
+  BindStateToController,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../../utils/initialization-utils';
+import {Button} from '../../../common/button';
+import {Hidden} from '../../../common/hidden';
+import {Bindings} from '../../atomic-search-interface/atomic-search-interface';
+import ArrowBottomIcon from 'coveo-styleguide/resources/icons/svg/arrow-bottom-rounded.svg';
+import {InitPopoverEventPayload} from './popover-event';
+
+/**
+ * @internal
+ * The `atomic-popover` displays a popover menu view of any facet
+ * @part value-button - The button used to display the popover menu
```

Comment:
```suggestion
 * @part value-button - The button to click to display the popover menu.
```
Also, I'm curious: why not `button` rather than`value-button`?
And I'm also wondering the same thing about `label` vs `value-label` 🤔

Reply:
I should probably call them popover-button and label. Maybe that would make it more clear?